### PR TITLE
Add color and rgb screens

### PR DESCRIPTION
**Pull request by Pablo756**

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import { Text, StyleSheet, TouchableOpacity } from "react-native";
 
-export const Button = ({ title, onPress }) => {
+export const Button = ({ title, onPress, _disabled }) => {
   return (
-    <TouchableOpacity style={styles.ct} onPress={onPress}>
+    <TouchableOpacity style={styles.ct} onPress={onPress} disabled={_disabled}>
```
Comment (polusik9393) [inline src/components/Button.js]: _disabled - remove _

File: src/components/RgbItem.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  disabledInc,
+  titleDec,
+  onPressDec,
+  disabledDec,
+}) => {
+  return (
+    <>
+      <Text style={styles.text}>Red: {color}</Text>
+      <Button title={titleInc} onPress={onPressInc} _disabled={disabledInc} />
+      <Button title={titleDec} onPress={onPressDec} _disabled={disabledDec} />
```
Comment (polusik9393) [inline src/components/RgbItem.js]: and here

File: src/screens/ColorScreen.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  return (
+    <View style={styles.ct}>
+      <Button
+        title={"Add Color"}
+        onPress={() => setColorData([...colorData, randomRgb()])}
+      />
+      <FlatList
+        style={{ marginVertical: 10 }}
+        contentContainerStyle={{ marginBottom: 20 }}
```
Comment (polusik9393) [inline src/screens/ColorScreen.js]: remove inline styles here

**Review by polusik9393 — CHANGES_REQUESTED** [review]

File: src/components/Button.js (hunk 1 above)
Comment (Pablo756) [reply src/components/Button.js]: done

**Review by Pablo756 — COMMENTED** [review]
done

File: src/components/RgbItem.js (hunk 2 above)
Comment (Pablo756) [reply src/components/RgbItem.js]: done

File: src/screens/ColorScreen.js (hunk 3 above)
Comment (Pablo756) [reply src/screens/ColorScreen.js]: done

**Pull request merged**